blocks/library/freeform/freeform-block.js

```diff
@@ -405,6 +405,7 @@ export default class FreeformBlock extends Component {
 			</BlockControls>,
 			<TinyMCE
 				key="editor"
+				className={ className }
```

Comment:
For styling purpose, but you're right it's not necessary here since it's not used in styling yet.

Though, I'd like us to favor these classnames over ".editor-visual-editor__block[data-type="core/freeform"]" for styling (when possible).